js/src/stats.js

```diff
@@ -54,6 +64,16 @@ module.exports = (common) => {
         expect(res).to.have.a.property('dataSent')
         expect(res).to.have.a.property('dupBlksReceived')
         expect(res).to.have.a.property('dupDataReceived')
+
+        expect(isNumber(res.provideBufLen)).to.eql(true)
+        expect(res.wantlist).to.be.an('array')
+        expect(res.peers).to.be.an('array')
```

Comment:
Would we consider this test to be correct if `res.peers` was a empty array?

Reply:
Yes and it might be empty in some cases.